indico/modules/events/layout/templates/layout_meeting_lecture.html

```diff
@@ -1,10 +1,24 @@
 {% extends 'layout/base.html' %}
-{% from 'forms/_form.html' import simple_form %}
+{% from 'forms/_form.html' import simple_form, form_rows %}
 
 {% block title %}
     {%- trans %}Layout{% endtrans -%}
 {% endblock %}
 
 {%- block content %}
-    {{ simple_form(form, save_reminder=true, back_button=false) }}
+    {% call simple_form(form, save_reminder=true, back_button=false) %}
+         <div>
```

Comment:
ah ok probably because it's not in a fieldset.. even though you could see whether there's any css selector you can use in the `nextAll()` call to stop it from removing the save button. maybe `:not(.form-group-footer)`